aws-s3/src/main/scala/busymachines/pureharm/aws/s3/package.scala

```diff
@@ -51,4 +51,7 @@ package object s3 {
 
   object S3DownloadURL extends PhantomType[String]
   type S3DownloadURL = S3DownloadURL.Type
+
+  object S3KeyPrefix extends PhantomType[String]
```

Comment:
Bad name, not sure what it means. Isn't it just a path that is expected to point to a folder?

Reply:
Yes, good point. I'm going to change it